examples/iket/all_features.py

```
@@ -0,0 +1,99 @@
+                        C[idx] = value
+                        T.iket.mark("store_done")
+                T.iket.mark("block_exit", payload=T.iket.payload(bx, dtype="int32"))
+
+    return main
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--iket-output-dir",
+        default=os.environ.get("TL_IKET_OUTPUT_DIR", "/tmp/tilelang_iket_all_features"),
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/tilelang_iket_all_features"

(S108)